apps/mobile/src/screens/home/home.tsx

```diff
@@ -1,15 +1,67 @@
-import React from 'react'
+import React from "react";
+import { Button, Card, Icon, ThemeProvider, Histories, } from "ui";
+import { useScanController } from "./hooks/use-scan-controller";
+import { DefaultLayout } from "../../shared/layouts/default-layout";
+import { useVisitData } from "./hooks/use-visit-data";
+import { Text } from 'react-native'
+import { CameraView } from 'expo-camera'
 
-import { JoinCard } from './components/join-card'
-import { DefaultLayout } from '../../shared/layouts/default-layout'
 
-/**
- * The home screen of the application.
- */
 export function HomeScreen() {
```

Comment:
There is absolutely no reason to change the `HomeScreen`. The version you put is outdated of few months.